website/docs/reference/commands/clone.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "About dbt clone command"
+sidebar_label: "clone"
+id: "clone"
+---
+
+The `dbt clone` command clones selected nodes from the specified state to the target schema(s). This command makes use of the `clone` materialization:
+- If your data platform supports zero-copy cloning of tables, and this model exists as a table in the other environment, dbt will create it in your environment as a clone
+- Otherwise, dbt will create a simple pointer view (`select * from` the other object)
+
+Note: 
+- The state to clone from is based on the location of nodes in the manifest provided to `--state`.
```

Comment:
I'm going to try deleting this note altogether in favor of linking to the relevant docs. See [here](https://github.com/dbt-labs/docs.getdbt.com/pull/3742/files#r1263903691).

We can always add this note (or something similar) back if needed.

```suggestion
```